src/components/Reports/StaffExpenseReport/SettingsDialog/SettingsDialog.test.tsx

```diff
@@ -0,0 +1,485 @@
+import React from 'react';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { DateTime } from 'luxon';
+import { SnackbarProvider } from 'notistack';
+import { DateRange } from '../Helpers/StaffReportEnum';
+import { Filters, SettingsDialog } from './SettingsDialog';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key,
+  }),
+}));
+
+jest.mock('src/components/common/DateTimePickers/CustomDateField', () => ({
```

Comment:
I'd like to avoid mocking this as well. I think you just need to wrap everything in `<LocalizationProvider dateAdapter={AdapterLuxon}>`. I'd recommend using the `TestComponent` pattern here so that you can add the provider in one place. You can look at other test files for examples of what I mean, or just ask, and I'd be happy to explain.